api/client/block/client_test.go

```diff
@@ -4,135 +4,134 @@
 package block_test
 
 import (
+	"github.com/golang/mock/gomock"
 	"github.com/juju/errors"
 	jc "github.com/juju/testing/checkers"
 	gc "gopkg.in/check.v1"
 
-	basetesting "github.com/juju/juju/api/base/testing"
+	basemocks "github.com/juju/juju/api/base/mocks"
 	"github.com/juju/juju/api/client/block"
 	apiservererrors "github.com/juju/juju/apiserver/errors"
 	"github.com/juju/juju/rpc/params"
 	"github.com/juju/juju/state"
-	coretesting "github.com/juju/juju/testing"
 )
 
 type blockMockSuite struct {
-	coretesting.BaseSuite
-	blockClient *block.Client
 }
 
 var _ = gc.Suite(&blockMockSuite{})
 
 func (s *blockMockSuite) TestSwitchBlockOn(c *gc.C) {
+	ctrl := gomock.NewController(c)
 	called := false
```

Comment:
"called" is no required. It's implicit with the go mock EXPECT() line.

Reply:
Yeah that's why I was doing a `DoAndReturn`, good call, thanks! :+1: